makevsix.js

```diff
@@ -0,0 +1,74 @@
+// 
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+require('shelljs/global');
+
+var path = require('path'),
+    fs = require('fs');
+
+var argv = require('yargs')
+    .usage('Usage: $0 <command>')
+    .command('create', 'Create VSIX')
```

Comment:
we could just add a second param that is "production" or "test" switch the overrides on that.